docs/services/files.rst

```diff
@@ -34,25 +50,63 @@ Creates a container::
     rack files container create --name <containerName> [optional flags]
     (echo containerName1 && echo containerName2) | rack files container create --stdin name
 
+**Response**
+
+.. code::
+
+    $ rack files container create --name RackCLI
+    Successfully created container [RackCLI]
+
 ``get``
 ^^^^^^^
-Retrieves a list of containers::
+Retrieves details on a specified container::
 
     rack files container get --name <containerName> [optional flags]
     (echo containerName1 && echo containerName2) | rack files container get --stdin name [optional flags]
 
+**Response**
+
+.. code::
+
+    $ rack files container get --name RackCLI
+    Name		RackCLI
+    ObjectCount	0
+    BytesUsed	0
+    ContentLength	0
+    AcceptRanges	bytes
+    ContentType	application/json; charset=utf-8
+    Read
+    Write
+    TransID		txc108b4d4ad9040898a210-0055c8d7dfdfw1
+    VersionsLocation
+
+
 ``update``
 ^^^^^^^^^^
 Create or update read and write permissions for a specified container::
 
     rack files container update --name <containerName> [optional flags]
 
+**Response**
+
+.. code::
+
+    $ rack files container update --name RackCLI
+    Successfully updated container [RackCLI]
+
+
 ``delete``
 ^^^^^^^^^^
 Permanently removes the specified container::
 
     rack files container delete --name <containerName> [optional flags]
-    (echo containerName1 && echo containerName2) | rack files container delete --stdin name [optional flags]
+    (echo containerName1 && echo containerName2) | rack files container delete --stdin name [optional flags
+
+.. code::
+
+    $ rack files container delete --name RackCLI
+    Running containers.Delete
```

Comment:
Thank you for pointing this out. This should be removed; I used it for debugging.